pypdf/filters.py

```diff
@@ -865,11 +886,22 @@ def bits2byte(data: bytes, size: Tuple[int, int], bits: int) -> bytes:
                         # gray lookup does not work : it is converted to a similar RGB lookup
                         lookup = b"".join([bytes([b, b, b]) for b in lookup])
                         mode = "RGB"
+                    # TODO : this is a work around until PIL is able to process CMYK images
```

Comment:
```suggestion
                    # The following is a work around until PIL is able to process CMYK images:
                    # https://github.com/python-pillow/Pillow/issues/7309
```

Reply:
Can you keep this TODO to keep some track where to do the upgrade when pillow will be upgraded

Reply:
I've added the TODO back, but I would like to have the URL of the issue in there. This way it's clearer to readers when this TODO wil dissappear (although one of us will very likely handle it :smile: )

Reply:
call me one ??? 🤣🤣🤣